embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/config/behavior/SessionBehavior.kt

```diff
@@ -24,9 +24,9 @@ interface SessionBehavior {
     fun isGatingFeatureEnabled(): Boolean
 
     /**
-     * Whether session control is enabled, meaning that features should be gated.
+     * Whether session control is disabled, meaning that can't end sessions manually.
      */
-    fun isSessionControlEnabled(): Boolean
```

Comment:
We can keep this as enabled at this level (app usage) so it matches the other kill switches. It's at the Config level that we should make this be identify itself as disabled